Update sso user retrieval (#692)
ubccr/xdmod: docs/simpleSAMLphp.md

*    `organization`
     * This will be used to identify which Organization the user should be associated with via the `modw.organization.name` column

To get a better idea of how these properties translate into expected behavior during SSO login. Please refer to the table below:

|SAML Organization Present|Person Present|Expected Behavior |
|:-----------------------:|:------------:|:-----------------|
| true                    | true         | The user is associated with the the Person's organization if found, else the SAML Organization if found, else the Unknown Organization|
| true                    | false        | The user is associated with the SAML Organization if found, else the Unknown organization|
| false                   | true         | The user is associated with the Persons organization if found, else the Unknown organization|
| false                   | false        | The user is associated with the Unknown organization|

> Here is an example `authsources.php` file:

Reviewer: ```suggestion
Here is an example `authsources.php` file **note: keys 60 and 61 must be present to validate the existence of username and organization, otherwise unexpected errors may occur**:
```

*    `organization`
     * This will be used to identify which Organization the user should be associated with via the `modw.organization.name` column

To get a better idea of how these properties translate into expected behavior during SSO login. Please refer to the table below:

|SAML Organization Present|Person Present|Expected Behavior |
|:-----------------------:|:------------:|:-----------------|
| true                    | true         | The user is associated with the the Person's organization if found, else the SAML Organization if found, else the Unknown Organization|
| true                    | false        | The user is associated with the SAML Organization if found, else the Unknown organization|
| false                   | true         | The user is associated with the Persons organization if found, else the Unknown organization|
| false                   | false        | The user is associated with the Unknown organization|

> Here is an example `authsources.php` file **note: keys 60 and 61 must be present to validate the existence of username and organization, otherwise unexpected errors may occur**:

Reviewer 2: Suggest using more precise language here: "otherwise unexpected errors may occur" is not a very useful statement to make. How about:
**note: keys 60 and 61 must be present**. These check that the _mandatory_ username and organization  properties are present.

> Here is an example `authsources.php` file **note: keys 60 and 61 must be present to validate the existence of username and organization, otherwise unexpected errors may occur**:

Reviewer: since @jpwhite4 didnt use suggestion ;)
```suggestion
Here is an example `authsources.php` file **note: keys 60 and 61 must be present** These check that the _mandatory_ username and organization properties are present.
```
